safeway-coupons

```diff
@@ -72,9 +73,7 @@ user_agent = ('Mozilla/5.0 (Windows NT 6.1; WOW64; rv:64.0) '
               'Gecko/20100101 Firefox/64.0')
 js_req_headers = {
     'Content-Type': 'application/json',
-    'DNT': '1',
     'Host': 'www.safeway.com',
-    'Origin': 'http://www.safeway.com',
```

Comment:
This was deleted to address a separate issue, perhaps CORS. The response to the coupon list request was empty when this header was set.